cmd/mimir/help-all.txt.tmpl

```diff
@@ -1983,6 +1983,8 @@ Usage of ./cmd/mimir/mimir:
     	Minimum TLS version to use. Allowed values: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13. If blank, the Go TLS minimum version is used.
   -shutdown-delay duration
     	[experimental] How long to wait between SIGTERM and shutdown. After receiving SIGTERM, Mimir will report not-ready status via /ready endpoint.
+  -store-gateway.ignore-native-histograms-on-read
```

Comment:
[nit] I would recommend positive config options, like `-store-gateway.native-histograms-enabled` which defaults to `false`.

Reply:
I can see the wording is spread in the whole changeset. Ignore this comment. Too late to change it now :)